### ISSUE TITLE
Error when running R CMD - problem in test?

### ISSUE THREAD

**Issue by TankredO** [florianhartig/BayesianTools#46]
```R
# plot prior predictive distribution and prior predictive simulations
plotTimeSeriesResults(sampler = out, model = createPredictions, observed = referenceData[,1], error = createError, prior = T, main = "Prior predictive")
#Error in plotTimeSeriesResults(sampler = out, model = createPredictions,  : 
#  T used instead of TRUE
```

**Comment by stefan-paul** [comment]
I don't really know why but in examples you always have to use TRUE/FALSE instead of T/F.
If not you'll get the error message above.

**Comment by TankredO** [comment]
Thank you! Solved with 1232f73

**Comment by florianhartig** [comment]
Simply CRAN rules
